kms/snippets/create_key_asymmetric_decrypt.py

```diff
@@ -11,9 +11,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 
+from google.cloud import kms
```

Comment:
Good point. I'm also happy to clear up the comments to make sure the commentary matches up. 

This and some other files get `isort`ed with the inline imports. They are currently 'out of order' (datetime stdlib coming after thirdparty) but the CI linting doesn't complain. Should these also get adjusted, since I'm making comment changes?  I worry that someone running isort in the future might get tripped up by this

Reply:
Thanks for asking. I don't see this mentioned in any official guidelines, but my preference would be to order them properly (similarly to GoLang: stdlibs first, third parties afterwards) - [see here for an example](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/compute/client_library/snippets/instance_templates/create_with_subnet.py#L23-L29).

Reply:
I've updated all examples to include imports outside of the methods, within region tags. Closing all associated discussions.